### PR TITLE
Sync old Position Attribute formatters to Modern

### PR DESCRIPTION
**Pull request by jinzo**
Hi,

this synces how the PositionAttribute formatting is done on the Modern interface, so that it matches the old interface. 
I extended the attributes which get formatted with speed and distance formatter, added voltage, driver unique id, consumption and volume.

### CODE REVIEW

File: modern/src/common/util/formatter.js [hunk 1]
```diff
@@ -64,6 +69,16 @@ export const formatNumericHours = (value, t) => {
   return `${hours} ${t('sharedHourAbbreviation')} ${minutes} ${t('sharedMinuteAbbreviation')}`;
 };
 
+export const formatDriverUniqueId = (value) => {
```
Comment (tananaev) [inline modern/src/common/util/formatter.js:72]: I don't know about this one. We only do light formatting in these functions. A lookup like this should probably be in a different place.

Comment (jinzo) [reply modern/src/common/util/formatter.js:72]: Was thinking the same initially, but then saw that it's implemented like that in the old interface and just decided to duplicate it. But I'm open to discuss how we can better represent driver(s)/mapping and just remove it from this PR/commits.

Comment (jinzo) [reply modern/src/common/util/formatter.js]: Will tackle this at some later time then, removed it for now. Thanks for the guidance.

**Comment by tananaev** [comment]
Merged, thanks.

**Pull request merged**